cumulusci/cli/cci.py

```diff
@@ -1152,21 +1171,28 @@ def org_shell(runtime, org_name, script=None, python=None):
     org_config.refresh_oauth_token(runtime.keychain)
 
     sf = get_simple_salesforce_connection(runtime.project_config, org_config)
+    printer = type(copyright)
+
+    helpers = repl_helpers(sf)
+
     globals = {
         "sf": sf,
         "org_config": org_config,
         "project_config": runtime.project_config,
+        "cci": printer("cci", org_shell_cci_help_message),
```

Comment:
I think people will get confused about the difference between `cci` at a shell prompt and `cci` inside the cci prompt. What about using `help` instead (i.e. make it be an object whose `__repr__` prints this message, but passes through to the help builtin when called)?

Reply:
bc673a8e